packages/flutter_tools/lib/src/android/gradle_utils.dart

```diff
@@ -75,6 +75,9 @@ RegExp(
     multiLine: true
 );
 
+final RegExp jellyBeanMinSdkVersionMatch =
```

Comment:
Consider adding a dart doc covering what this regex is expecting to match in more plain text.

Reply:
Added a comment